app/javascript/react/screens/App/Overview/screens/MappingWizard/components/MappingWizardClustersStep/MappingWizardClustersStep.js

```diff
@@ -12,6 +18,13 @@ class MappingWizardClustersStep extends React.Component {
       mappings: [], // eslint-disable-line react/no-unused-state
```

Comment:
let's go ahead call these `clusterMappings` and `selectedClusterMappings`... will be reusing those names in next step.

Reply:
I was planning on lifting `mappings` from `MappingWizardClustersStep` and into the `redux-form` part of the store.  Def interested in your thoughts

![fullscreen_2_19_18__8_39_am](https://user-images.githubusercontent.com/15141412/36380554-b411fdce-1550-11e8-8046-30027c3ce0bd.png)

Reply:
Here is how I was planning on integrating redux-form

https://github.com/michaelkro/miq_v2v_ui_plugin/compare/mapping-wizard-clusters-step...clusters-step-redux-form